Contributing.md

```diff
@@ -0,0 +1,15 @@
+### One issue or bug per Pull Request
+
+Keep your Pull Requests small. Small PRs are easier to reason about which makes them significantly more likely to get merged.
+
+### Issues before features
```

Comment:
I’m actually ok with not enforcing this as a rule. I’d rather make this a suggestion. If someone wants to put up a PR to demonstrate an idea, I think that’s ok.